cura/Scene/CuraSceneNode.py

```diff
@@ -72,9 +78,32 @@ def getDiffuseColor(self) -> List[float]:
             1.0
         ]
 
+    def collidesWithBbox(self, check_bbox):
```

Comment:
Add documentation for this method

Reply:
In particular it's interesting to know what happens in edge cases when they exactly align. This is important for the whole 214.9mm vs. 215mm build volume thing.